From: Ali Erdinc Koroglu <ali.koroglu@oss.qualcomm.com>
Date: Wed, 5 Aug 2026 12:40:24 +0300
Subject: build: name the package sensinghub and bump to the current version

AC_INIT declared the package as sensing-hub at version 1.0.0. Neither has been
touched since the first release, so PACKAGE_VERSION, VERSION in config.h and the
make dist tarball name have been wrong for every version since, and the
autotools-facing name disagreed with the rest of the project: the repo name and
the library names use the unhyphenated form.

Use sensinghub and bump the version to 2.2.2. The LT_* macros are deliberately
left alone: they feed -version-number and should move on ABI changes, not on
releases.

Signed-off-by: Ali Erdinc Koroglu <ali.koroglu@oss.qualcomm.com>
---
 Makefile.am                           | 2 +-
 configure.ac                          | 6 +++---
 sensing-hub.pc.in => sensinghub.pc.in | 0
 3 files changed, 4 insertions(+), 4 deletions(-)
 rename sensing-hub.pc.in => sensinghub.pc.in (100%)

diff --git a/Makefile.am b/Makefile.am
index 814a7d6..e388b95 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -9,6 +9,6 @@ dirs = apis/proto \
 SUBDIRS = $(dirs)
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = sensing-hub.pc
+pkgconfig_DATA = sensinghub.pc
 
 EXTRA_DIST = $(pkgconfig_DATA) README.md LICENSE.txt
diff --git a/configure.ac b/configure.ac
index 4ee7cd1..62a48df 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,8 +5,8 @@
 # Requires autoconf tool later than 2.61
 AC_PREREQ(2.61)
 
-# Initialize the sensors package version 1.0.0
-AC_INIT([sensing-hub], 1.0.0)
+# Initialize the sensors package version 2.2.2
+AC_INIT([sensinghub], 2.2.2)
 
 # Does not strictly follow GNU Coding standards
 AM_INIT_AUTOMAKE([foreign subdir-objects dist-bzip2 tar-pax])
@@ -108,7 +108,7 @@ AC_CONFIG_FILES([             \
         examples/SessionClient/Makefile \
         utils/Makefile \
 		sessionlogger/Makefile  \
-        sensing-hub.pc
+        sensinghub.pc
                ])
 AC_OUTPUT
 
diff --git a/sensing-hub.pc.in b/sensinghub.pc.in
similarity index 100%
rename from sensing-hub.pc.in
rename to sensinghub.pc.in
